pages/admin/content/edit.jsx

```diff
@@ -1,23 +1,36 @@
 import React from 'react';
+
 import { load, save } from '../../../core/client/api/content';
 import { getContentTypeMeta } from '../../../core/entities/content-type';
+import { getFieldComponentByType } from '../../../core/entities/field-type';
 import ContentTypeForm from '../../../core/ui/ContentTypeForm';
 
-const Edit = ({ content, typeSchema }) => (
-  <ContentTypeForm
-    initialValues={content}
-    typeSchema={typeSchema}
-    onSave={values => {
-      save(content.type, values.slug, values);
-    }}
-  />
-);
+const Edit = ({ content, typeSchema }) => {
+  return (
+    <ContentTypeForm
+      initialValues={content}
+      typeSchema={{
+        ...typeSchema,
+        fields: typeSchema.fields.map(field => ({
+          ...field,
+          Renderer: getFieldComponentByType(field.type),
```

Comment:
In the future, we should consider allowing to plug in custom fields in the user app. Let's don't do anything yet and just be aware and keep thinking about the approach in advance. we can discuss tomorrow if there are any questions